src/main/java/com/ibm/crypto/plus/provider/XDHKeyPairGenerator.java

```diff
@@ -119,7 +119,7 @@ private void initializeImpl(NamedParameterSpec params) {
     @Override
     public KeyPair generateKeyPair() {
         try {
-            int pub_size = CurveUtil.getPublicCurveSize(serviceCurve);
+            int pub_size = CurveUtil.getCurveSize(serviceCurve);
```

Comment:
Should we change the name of the variable to `keySize`?

Reply:
Yes.